key-terms.md

```diff
@@ -0,0 +1,135 @@
+Key Terms
+---------
+
+Of the 83 glossary terms used in WCAG 2.2 there are two key glossary terms that need to be interpreted significantly differently when applied to non-web ICT. These are: “content” and “user agent”. Further, the glossary term “Web page” in WCAG 2.2 is replaced with newly defined terms “document” and “software”, and both “set of web pages” and “multiple web pages” are replaced with the newly defined terms “set of documents” and “set of software programs”. Finally, as part of addressing the fact that non-Web software doesn't leverage the WCAG 2.2 notion of a user agent, we introduce the new term “accessibility services of platform software”. The remaining 79 glossary terms from WCAG 2.2 are addressed in [Chapter 7 Comments on Definitions in WCAG 2.2 Glossary in Appendix A](http://w3c.github.io/wcag2ict/#comments-on-definitions-in-wcag-2-0-glossary-in-appendix-a). Terms defined and used in WCAG2ICT are applicable only to the interpretation of the guidance in this document. The particular definitions should not be interpreted as having applicability to situations beyond the scope of WCAG2ICT. Further information on usage of these terms follows.
+
+### Accessibility Services of Platform Software
+
+The term **accessibility services of platform software**, as used in WCAG2ICT, has the meaning below:
+
+accessibility services of platform software (as used in WCAG2ICT)
+
+services provided by an operating system, user agent, or other platform software that enable non-web documents or software to expose information about the user interface and events to assistive technologies and accessibility features of software
+
+These services are commonly provided in the form of accessibility APIs (application programming interfaces), and they provide two-way communication with assistive technologies, including exposing information about objects and events.
+
+### Content (on and off the Web)
+
+WCAG 2.2 defines CONTENT as:
+
+For non-web content it is necessary to view this a bit more broadly. Within WCAG2ICT, the term “content” is used as follows:
+
+content (non-web content) (as used in WCAG2ICT)
+
+information and sensory experience to be communicated to the user by means of [software](http://w3c.github.io/wcag2ict/#wcag2ict-def_software "WCAG2ICT Definition: software"), including code or markup that defines the content's [structure](http://w3c.github.io/wcag2ict/#wcag2ict-def_structure "WCAG2ICT Definition: structure"), [presentation](http://www.w3.org/TR/WCAG22/#dfn-presentation "WCAG Definition: presentation"), and interactions
```

Comment:
Also, the rollover info for definitions will likely not work with this without some additional work.

Reply:
We will need to sort out how we want terms to work, right now they're not definitions as such in WCAG2ICT. We either can import from WCAG, or redefine with markup in WCAG2ICT. Then the rollover will work again.